src/cryptsetup/cryptsetup-generator.c

```diff
@@ -29,6 +29,8 @@ typedef struct crypto_device {
         char *uuid;
         char *keyfile;
         char *keydev;
+        char *hdrdev;
```

Comment:
not a fan of needless abbreviations, please call this "headerdev" or so

Reply:
Fixed